Test/allocated1/dafny0/ControlStructures.dfy.expect

```diff
@@ -8,8 +8,8 @@ ControlStructures.dfy(218,17): Error: assertion might not hold
 ControlStructures.dfy(235,20): Error: assertion might not hold
 ControlStructures.dfy(238,29): Error: assertion might not hold
 ControlStructures.dfy(241,16): Error: assertion might not hold
-ControlStructures.dfy(350,2): Error: cannot prove termination; try supplying a decreases clause for the loop
-ControlStructures.dfy(381,2): Error: cannot prove termination; try supplying a decreases clause for the loop
+ControlStructures.dfy(350,2): Error: cannot prove termination; try supplying a decreases clause
```

Comment:
Can you put back the message "for the loop"? I'm not sure but I feel it was better with it.